huxley/www/static/js/huxley/stores/DelegateStore.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+ * Use of this source code is governed by a BSD License (see LICENSE).
+ */
+
+'use strict';
+
+var $ = require('jquery');
+var Promise = require('es6-promise').Promise;
+var Dispatcher = require('../dispatcher/Dispatcher');
+var {Store} = require('flux/utils');
+
+
+var _delegatePromise = null;
```

Comment:
Since we're passing in a school id, we should actually have an object of promises per ID:
```js
var _delegatePromises = {};
...
if (!_delegatePromises[schoolID]) {
  _delegatePromises[schoolID] = new Promise(...);
}
```
Otherwise, no matter what schoolID you pass in, you'll get the delegates from the first school!